core/monad_properties.py

```diff
@@ -186,6 +187,32 @@ def get_settings(self):
     subtype: EnumProperty(items=float_items, name="Subtype", default='NONE')
 
 
+
+def find_hightest_num_in_matching_params(obj_id, kind):
+    regex = "^(?P<kind>ints|floats)_(?P<number>\d+)_(?P<variable>\S+)$"
+    nums = [1]
+    for name in obj_id.__annotations__:
+        matches = re.search(regex, name)
+        if matches and matches.group('kind') == kind:
+            nums.append(int(matches.group('number')))
+    return max(nums)
+
+
+def ensure_unique(obj_id, new_prop_name):
+
+    # can be used unchanged
+    if not (new_prop_name in obj_id.__annotations__):
+        return new_prop_name
```

Comment:
Why don't use `not in` operator?

Reply:
you mean 
```python
if new_prop_name not in annotations:
        return new_prop_name
```
i'll think about it. but it's a minor issue